Tools/autotest/arduplane.py

```diff
@@ -1745,6 +1745,16 @@ def fly_soaring(self):
 
         self.progress("Mission OK")
 
+    def fly_terrain_mission(self):
```

Comment:
You might consider using 

https://github.com/ardupilot/ardupilot/blob/wip%2Fhandle-command-do-aux-function/Tools/autotest/arduplane.py#L515

You can see the MissionTest actually just uses a lambda to do it ('though you currently have the "wipe", so if that's required you could use that.  Still, would turn your body into a two-liner.

Reply:
I also need to arm - I could add "self.arm_and_fly_mission" if this is useful